packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1048,9 +1048,10 @@ oie.consent.scopes.enduser.title = would like to:
 oie.consent.enduser.accept.label = Yes, it's me
 oie.consent.enduser.deny.label = No, it's not me
 oie.consent.enduser.title = Did you just try to sign in?
-oie.consent.enduser.email.allow.description = To continue, return to the device or window where you requested the email link. You may close this window at any time.
-oie.consent.enduser.deny.description = You may close this window at any time.
-oie.return.to.original.tab = Please return to the original tab.
+oie.consent.enduser.email.allow.title = Success! Return to the original tab or window
```

Comment:
Looks like we reverted this back to a new key.

Reply:
New key is required for `title` of the page.

Reply:
this change will go with monthly release PR so i think it should be okay to add new key.